utils_nlp/models/transformers/common.py

```diff
@@ -4,16 +4,18 @@
 # This script reuses some code from
```

Comment:
There were recent changes in staging that you might want to merge here.

Reply:
merged.